templates/includes/gtm-datalayer.html

```diff
@@ -0,0 +1,6 @@
+<script>
+    window.dataLayer = window.dataLayer || [];
+    {% if data_json %}
+    window.dataLayer.push({{ data_json|safe }});
```

Comment:
I think I have a preference for [`json_script`](https://github.com/django/django/blob/7119f40c9881666b6f9b5cf7df09ee1d21cc8344/django/utils/html.py#L62) here.

Some records properties contain markup (see `C1383853`) and I can see a scenario where the `dataLayer` and  is broken because we're not escaping `<`, `>`, or `&`.

Worse-case scenario testing here, I've manually set:

```python
self.availability_delivery_condition = '</script><script>window.alert(window);</script>'
```
<img width="1413" alt="Screenshot 2022-03-04 at 11 25 31" src="https://user-images.githubusercontent.com/535833/156756011-6d29a65a-50e4-4cf9-bb83-1cc32fbb4ec9.png">

Reply:
Cheers @danbentley. From what I can tell, we can do all of the same escaping that `json_script` does, without the complication of using multiple script tags or Django tags/filters. Would you be okay with a fix that addressed things in that way?